### DISCUSSION TITLE
Import data from excel [Jun 02]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2314] [Discord Threads]
> 🙋‍♂️ **Dran**  
## Hello, I am new here, I am thinking in using NocoDB, but there is a feature I need very badly, is it possible to import from excel? like here in Airtable? Speciall the capacity to import many times on top of an existing base:  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/981556946945593384/unknown.png)

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@724109247167725588>   
####    
#### NocoDB currently supports creating a new table from excel (by either uploading it or specifying its URL). This will create a new table, create columns & import records.   
####    
#### You can import many times on the same project. Each import will create a new table though.  
![image0](https://cdn.discordapp.com/attachments/981556947570556988/981558682313711687/Screenshot_2022-06-01_at_7.33.26_PM.png)  
#  
> 🙋‍♂️ **Dran**  
#### thank you  
#  
> 🙋‍♂️ **Dran**  
#### Can I PASTE excel files on top of a base having the same columns?  
#  
> 👨‍🦲 **dstala**  
#### Currently, we support (beta feature) uploading records from a CSV.  
![image0](https://cdn.discordapp.com/attachments/981556947570556988/981565216787693589/Screenshot_2022-06-01_at_7.59.19_PM.png)  
#  
> 🙋‍♂️ **Dran**  
#### ohhh I managed to open a free Heroku and installed Noco. I'll check by myself THANK you  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/981556947570556988)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
